src/trans-word-level/instantiate_word_level.cpp

```diff
@@ -428,6 +429,45 @@ wl_instantiatet::instantiate_rec(exprt expr, const mp_integer &t) const
     DATA_INVARIANT(no_timeframes != 0, "must have timeframe");
     return {no_timeframes - 1, conjunction(conjuncts)};
   }
+  else if(expr.id() == ID_sva_ranged_s_eventually)
+  {
```

Comment:
Not that this change makes the decisive difference, but: `instantiate_rec` has become very very long and could do with breaking into smaller functions. Perhaps this opportunity could be taken to at least move this case to a function of its own?

Reply:
Will do. What bugs me most is the duplication between instantiate_word_level.cpp and property.cpp.